----- 04/11/21 14:51:54 -----
Traceback (most recent call last):
  File "/home/jenish/Desktop/demo/chess game simple/gui.py", line 174, in play_as
    game = make_AI_move(game, color)
  File "/home/jenish/Desktop/demo/chess game simple/gui.py", line 148, in make_AI_move
    new_game = chess.make_move(game, chess.get_AI_move(game, AI_SEARCH_DEPTH))
  File "/home/jenish/Desktop/demo/chess game simple/chess.py", line 1340, in get_AI_move
    if find_in_book(game):
  File "/home/jenish/Desktop/demo/chess game simple/chess.py", line 1426, in find_in_book
    book_file = open("book.txt")
FileNotFoundError: [Errno 2] No such file or directory: 'book.txt'

Playing as BLACK:
	
	rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w KQkq - 0 1
-----------------------------

